### PR TITLE
Teleport 5.0 Quick Start

### PR DESCRIPTION
**Pull request by benarent**
Updated Teleport 5.0 Quickstart.
- [x] Simplified setup, I dropped `teleport configure` as it adding more complexity. It would be good to consider adding a type flag to configure. `--type=quickstart,node,app,kube` etc, that'll provide an optioned quick start example.
- [x] New section to focuses on TLS & DNS setup so customers don't have to run the gauntlet of `--insecure`
- [x] Added examples and single liners for `teleport start` for both Applications and Servers.

( still needs a bit of testing but ready for review )

### CODE REVIEW

File: docs/5.0/quickstart.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: The quick start guide for how to set up modern SSH access to cloud
 # Teleport Quick Start
 
 This tutorial will guide you through the steps needed to install and run
-Teleport on a single node, which could be your local machine but we recommend a
+Teleport Unified Access Plane, which could be your local machine but we recommend a
```
Comment (klizhentas) [inline docs/5.0/quickstart.md:9]: why do we recommend a vm?

Comment (klizhentas) [inline docs/5.0/quickstart.md:9]: ~~unified access plane~~

Comment (benarent) [reply docs/5.0/quickstart.md:9]: removed

Comment (benarent) [reply docs/5.0/quickstart.md:9]: For the quickstart we decided to write it for Junior Sysadmins and SREs, from the work for Teleport User Profiles. I've removed the VM reference to simply make the setup about running Teleport on Linux Machines. 

> This tutorial will guide you through the steps needed to install and run Teleport on Linux machine(s).

In future vresions we'll also have a quickstart focused on SREs using a Kubernetes cluster.

File: docs/5.0/quickstart.md [hunk 2]
```diff
@@ -36,7 +75,32 @@ When setting up Teleport, we recommend running it with Teleports YAML configurat
...
+    ssh_service:
+        enabled: "yes"
+        labels:
+            env: staging
+    proxy_service:
+        enabled: "yes"
+        listen_addr: 0.0.0.0:3023
+        web_listen_addr: 0.0.0.0:3080
+        tunnel_listen_addr: 0.0.0.0:3024
+        https_keypairs: []
```
Comment (webvictim) [inline docs/5.0/quickstart.md]: If this is blank, does it just generate a self-signed cert? (same as not specifying `https_cert_file` and `https_key_file`) @russjones

File: docs/5.0/quickstart.md [hunk 3]
```diff
@@ -23,10 +22,50 @@ VM.
...
+
+=== "yum repo / Amazon Linux 2"
+
+    ```bash
+    yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
+    yum install teleport
+    ```
+    _Optional: Using Dandified YUM_
+    ```bash
+    dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
```
[the remark this answers is not in the document]
Comment (benarent) [reply docs/5.0/quickstart.md]: Cherry picked this comment.

File: docs/5.0/quickstart.md [hunk 4]
```diff
@@ -36,7 +75,32 @@ When setting up Teleport, we recommend running it with Teleports YAML configurat
 $ teleport configure > teleport.yaml
 ```
 
-??? example "Example teleport.yaml" We'll pre-populate your configuration with a nodename and a strong randomly generated static token. yaml --8<-- "snippets/install/teleport-config.yaml"
+!!! tip "Example teleport.yaml"
+
+    ```
+    teleport:
+        nodename: ip-172-31-13-139
```
[the remark this answers is not in the document]
Comment (benarent) [reply docs/5.0/quickstart.md]: This has been removed.

File: docs/5.0/quickstart.md [hunk 5]
```diff
@@ -23,10 +22,50 @@ VM.
 This guide is only meant to demonstrate how to run Teleport in a sandbox or demo
 environment, and showcase a few basic tasks you can do with Teleport.
 
-## Step 1: Install Teleport on a Remote VM
+## Step 1: Install Teleport on a Linux Host
 
-There are several ways to install Teleport. Take a look at the [Teleport Installation](installation.md)
-page to pick the method convenient for you.
+There are several ways to install Teleport. 
+Take a look at the [Teleport Installation](installation.md) page to pick the method convenient for you.
```
[the remark this answers is not in the document]
Comment (benarent) [reply docs/5.0/quickstart.md:28]: cherry picked this over to my branch.

**Pull request merged**